src/translator/quality_estimator.cpp

```diff
@@ -267,22 +267,4 @@ std::vector<SubwordRange> mapWords(const std::vector<float>& logProbs, const Ann
   return wordIndices;
 }
 
-std::vector<ByteRange> subwordToWords(const std::vector<SubwordRange>& wordIndices, const AnnotatedText& target,
-                                      const size_t sentenceIdx) {
-  std::vector<ByteRange> words;
-
-  for (const SubwordRange& wordIndice : wordIndices) {
-    size_t wordBegin = target.wordAsByteRange(sentenceIdx, wordIndice.begin).begin;
-    size_t wordEnd = target.wordAsByteRange(sentenceIdx, wordIndice.end).begin;
-
-    if (isspace(target.text.at(wordBegin))) {
-      ++wordBegin;
-    }
-
-    words.emplace_back(ByteRange{wordBegin, wordEnd});
-  }
-
-  return words;
-}
```

Comment:
What's happening here? There was a "words as meaningful QE units" notion before, are we losing this with the HTML integrated QE?

Reply:
The QualityEstimator class still works with this assumption, subwords are grouped into words in `mapWords`: https://github.com/browsermt/bergamot-translator/blob/bce30cfcb3079e3d9cf82ef005815e1ab467517f/src/translator/quality_estimator.cpp#L241-L268

The bit of code I removed here turned those groupings into a single byterange. That has been moved to the WASM bindings bit since that's the last possible moment to do it.

I think most clients will find it useful to do this themselves by accessing the AnnotatedText's data. But since we don't expose those to Javascript, and I didn't want to start a half-assed attempt at reworking the Javascript API, I moved the functionality above to here:
https://github.com/browsermt/bergamot-translator/blob/bce30cfcb3079e3d9cf82ef005815e1ab467517f/wasm/bindings/response_bindings.cpp#L33-L52